docs/how-we-work/tools/text-editors.md

```diff
@@ -1,17 +0,0 @@
-# Text editors
```

Comment:
I figured all the other Tools were SaaS offerings, and Editors are downloaded applications. Also, there is a strong overlap between Editors and Development Environments. But I can see it both ways.